cookbook/form/data_transformers.rst

```diff
@@ -38,17 +152,11 @@ for converting to and from the issue number and the ``Issue`` object::
 
     class IssueToNumberTransformer implements DataTransformerInterface
     {
-        /**
-         * @var ObjectManager
-         */
-        private $om;
+        private $em;
 
-        /**
-         * @param ObjectManager $om
-         */
-        public function __construct(ObjectManager $om)
+        public function __construct(ObjectManager $em)
```

Comment:
+1 - especially to using the longer `$entityManager` - that's free documentation in the code :)